### PR TITLE
New Haiku

### PR DESCRIPTION
**Pull request by AB-Yusuf**
I created a new haiku following the stated guidelines.
I added the stated format to the top of my md file.
I saved and named it "Note_to_self.md"

### CODE REVIEW

File: _haikus/Note_to_self.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: haiku
+title: Note to self
+author: Abubakar
+---
+
+
+In life being truthful
```
Comment (ltagliaferri) [inline _haikus/Note_to_self.md]: This line should have 5 syllables and should end with two spaces `  `

Maybe you can try

```
In life, being true  
```

File: _haikus/Note_to_self.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+to yourself requires patience
```
Comment (ltagliaferri) [inline _haikus/Note_to_self.md]: This should have 7 syllables and should end with two spaces, you can try:

```
to you requires patience
```

Comment (AB-Yusuf) [reply _haikus/Note_to_self.md]: Thank you @ltagliaferri i , I have made a new commit

**Comment by AB-Yusuf** [comment]
Can someone kindly take time out to review my pull request?
Thank you.

File: _haikus/Note_to_self.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+layout: haiku
+title: Note to self
+author: Abubakar
+---
+
+
+Listen, be yourself.
```
Comment (ltagliaferri) [inline _haikus/Note_to_self.md]: Please add a `<br>` tag or two spaces `  ` at the end of this line and the second line to force line breaks.

Comment (AB-Yusuf) [reply _haikus/Note_to_self.md]: I will do that.

Comment (AB-Yusuf) [reply _haikus/Note_to_self.md]: I have made the requested changes
Thank you for patiently reviewing my Haiku.

**Comment by ltagliaferri** [comment]
Thank you for your contribution to open source 🎉

**Pull request merged**